libs/service-portal/licenses/src/components/QRCodeModal/PkPass.tsx

```diff
@@ -30,131 +24,87 @@ export const PkPass = ({
   expireDate,
   textButton = false,
 }: PkPassProps) => {
-  const [pkpassQRCode, setPkpassQRCode] = useState<string | null>(null)
   const [pkpassUrl, setPkpassUrl] = useState<string | null>(null)
   const [generatePkPass] = useMutation(CREATE_PK_PASS)
-  const [generatePkPassQrCode, { loading: QRCodeLoading }] = useMutation(
-    CREATE_PK_PASS_QR_CODE,
-  )
   const { data: userProfile } = useUserProfile()
   const [displayLoader, setDisplayLoader] = useState<boolean>(false)
-  const [modalOpen, setModalOpen] = useState(false)
   const [isMobile, setIsMobile] = useState(false)
   const [fetched, setFetched] = useState(false)
+  const [linkError, setLinkError] = useState(false)
   const locale = (userProfile?.locale as Locale) ?? 'is'
   const { formatMessage } = useLocale()
   const { width } = useWindowSize()
   const timeFetched = new Date() // Used to compare if license is expired
```

Comment:
Do we need this? It seems like we could go without it. It's not really serving much of a purpose is it?